jest/verifyComponent.tsx

```diff
@@ -59,7 +59,7 @@ export function verifyComponent(
         role: 'MyRole',
         'data-testid': 'MyTestID',
       } as const;
-      render(<Test ref={ref} {...attrs} {...callbacks} />);
+      render(<Test ref={ref} {...attrs} />);
```

Comment:
`...callbacks` did nothing but throw a bunch of warnings in Jest. I'm not sure why it was there.